Dockerfile

```diff
@@ -10,6 +10,9 @@ RUN pip install --user \
 
 FROM python:3.7-stretch as service
 
+RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg  add - && apt-get update -y && apt-get install google-cloud-sdk -y
```

Comment:
omg bare ubuntu can't install google-cloud-sdk! `neuro` doesn't need this magic hehe

Reply:
I guess that's because `google-cloud-sdk` changes too rapidly for being included in default Ubuntu repos :)